collect_app/src/main/AndroidManifest.xml

```diff
@@ -135,9 +135,7 @@ the specific language governing permissions and limitations under the License.
             android:name=".formlists.blankformlist.BlankFormListActivity"
             android:launchMode="singleTop" />
         <activity android:name=".activities.FormDownloadListActivity" />
-        <activity
-            android:name=".activities.DeleteSavedFormActivity"
-            android:configChanges="orientation|screenSize" />
```

Comment:
This was previously how Delete Blank Form retained selections between config changes.